### PR TITLE
RESTWS-757, add includeNullFufillerStatus param to OrderSearchHandler

### PR DESCRIPTION
**Pull request by cioan**
@mogoodrich , here is the related web services ticket to the TRUNK-5682. Thanks!

### CODE REVIEW

**Comment by HerbertYiga** [comment]
am seing failing tests here!!

**Comment by cioan** [comment]
> am seing failing tests here!!

Yes, @HerbertYiga . [TRUNK-5682](https://github.com/openmrs/openmrs-core/pull/3054) needs to be merged in first.

**Comment by mogoodrich** [comment]
Thanks @cioan, I just merged in the TRUNK-5682.
Let's backport that fix to 2.3.x, and then change this commit to a new 2.3 package and OrderSearchHandler2_3... then we can upgrade the PIH EMR to 2.3....

**Comment by cioan** [comment]
@mogoodrich , I just commited the backporting to 2.3.x. Thanks for reviewing.

File: omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_3/OrderSearchHandler2_3.java [hunk 1]
```diff
@@ -90,11 +92,12 @@
 	            REQUEST_PARAM_CANCELED_OR_AUTO_EXPIRE_ON_OR_BEFORE_DATE,
 	            REQUEST_PARAM_ACTION,
 	            REQUEST_PARAM_FULFILLER_STATUS,
+	            REQUEST_PARAM_INCLUDE_NULL_FULFILLER_STATUS,
 	            REQUEST_PARAM_EXCLUDE_CANCELED_AND_EXPIRED,
 	            REQUEST_PARAM_INCLUDE_VOIDED).build();
 	
 	private final SearchConfig searchConfig = new SearchConfig("default", RestConstants.VERSION_1
-	        + "/order", Arrays.asList("2.4.*"), searchQuery);
+	        + "/order", Arrays.asList("2.3.*"), searchQuery);
```
Comment (mogoodrich) [inline omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_3/OrderSearchHandler2_3.java]: let's add 2.4.* back in here as well for future releases (I'll add that quickly in a post-commit, want to get this through and up on our build as soon as possible...)

Comment (cioan) [reply omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_3/OrderSearchHandler2_3.java]: Thanks @mogoodrich !

**Pull request merged**